src/data/user-list.ts

```diff
@@ -0,0 +1,65 @@
+import {ImageProps} from 'react-native';
+
+export interface UserList {
+  email: string;
+  password: string;
+  image: ImageProps['source'];
+}
+
+const lucasPicture = require('../assets/images/lucas-picture.jpeg');
+const matheusPicture = require('../assets/images/matheus-picture.jpeg');
+const alanPicture = require('../assets/images/alan-picture.jpeg');
+const victorPicture = require('../assets/images/victor-picture.jpeg');
+const viniciusPicture = require('../assets/images/vinicius-picture.jpeg');
+const juliaPicture = require('../assets/images/julia-picture.jpeg');
+const mariaPicture = require('../assets/images/maria-picture.jpeg');
+const talitaPicture = require('../assets/images/talita-picture.jpeg');
+const ericaPicture = require('../assets/images/erica-picture.jpeg');
+
+export const userList: UserList[] = [
+  {
+    email: 'email: lucas@taqtile.com.br',
+    image: lucasPicture,
```

Comment:
O bot especificava 2 campos apenas, não tinah imagem. Quando vc for integrar com o servidor, ele não vai te retornar uma imagem, então vc vai acabar não usando isso.

Reply:
só usei para ilustrar melhor a tarefa, na hora de integrar no servidor utilizo somente os campos de email e senha